inno_setup.iss

```diff
@@ -190,6 +193,11 @@ Root: HKCR; Subkey: "{#MyAppName}soepatch";                     ValueData: "Arch
 Root: HKCR; Subkey: "{#MyAppName}soepatch\DefaultIcon";         ValueData: "{app}\ArchipelagoSNIClient.exe,0";                           ValueType: string;  ValueName: ""; Components: client/sni
 Root: HKCR; Subkey: "{#MyAppName}soepatch\shell\open\command";  ValueData: """{app}\ArchipelagoSNIClient.exe"" ""%1""";                  ValueType: string;  ValueName: ""; Components: client/sni
 
+Root: HKCR; Subkey: ".apl2ac";                                 ValueData: "{#MyAppName}l2acpatch";        Flags: uninsdeletevalue; ValueType: string;  ValueName: ""; Components: client/sni
```

Comment:
I'd prefer to have https://github.com/ArchipelagoMW/Archipelago/pull/1262/ merged first, to make this section obsolete.